edm4eic.yaml

```diff
@@ -433,22 +433,16 @@ datatypes:
 
   edm4eic::Vertex:
     Description: "EIC vertex"
-    Author: "W. Armstrong, S. Joosten, based off EDM4hep"
+    Author: "J. Osborn"
     Members:
-      - int32_t             primary       // Boolean flag, if vertex is the primary vertex of the event
+      - int32_t             type       // Type flag, to identify what type of vertex it is (e.g. primary, secondary, generated, etc.)
       - float               chi2          // Chi-squared of the vertex fit
-      - float               probability   // Probability of the vertex fit
-      - edm4hep::Vector3f   position      // [mm] position of the vertex.
+      - int               ndf   // NDF of the vertex fit
+      - edm4hep::Vector4f   position      // [mm] position + time t0 of the vertex.
```

Comment:
```suggestion
      - edm4hep::Vector4f   position      // position [mm] + time [ns] of the vertex
```

Reply:
Do we want to store the time t0 in units of `mm`? That would imply `ct` and is used in some data models. We should be specific.

Reply:
That was never discussed in the working group discussions, so the short answer is I don't know. Is there any precedent elsewhere in our EDM to use units of `mm` for time? I would personally default to time since it is more fundamental, i.e. you can always convert to a distance assuming `ct` but if for whatever reason `c` was not the correct velocity you couldn't convert back to `t`

Reply:
I default to 4th only because that is what Acts does, which isn't necessarily a good reason but would offer consistency. I'll add a comment